### PR TITLE
LPS-188373 Sort Objects Relationships,Layouts and Actions entities by label by default

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
Task : [LPS-188373](https://liferay.atlassian.net/browse/LPS-188373)

### CODE REVIEW

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Comment by p-albuquerque** [comment]
Starting review

**Review by p-albuquerque — APPROVED** [review]

**Review by MarinhoFeliphe — APPROVED** [review]

**Comment by MarinhoFeliphe** [comment]
@pedro-oliveira446 it looks good to me!

please solve the conflicting files and rerun the ci tests.

Thank you!

**Review by alinedoleron — APPROVED** [review]
LGTM!

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Comment by pedro-oliveira446** [comment]
Hi @MarinhoFeliphe, @p-albuquerque,

I had to add five new commits to fix the related integration tests. Can you review again, please ?

**Review by mateussgn — APPROVED** [review]

**Comment by mateussgn** [comment]
LGTM! 🎉

**Comment by mateussgn** [comment]
Hey, @guilhermedcamacho, can double-check this PR review?

File: modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectActionResourceTest.java [hunk 1]
```diff
@@ -192,6 +215,48 @@ private ObjectAction _addObjectAction() throws Exception {
 			_objectDefinition.getObjectDefinitionId(), randomObjectAction());
 	}
 
+	private void _unsafeTriConsumer(
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectActionResourceTest.java:218]: @pedro-oliveira446 rename to `_getPageWithSortStringUnsafeTriConsumer`

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]
@pedro-oliveira446 see my coments above.

Thanks for your hard work!

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Review by MarinhoFeliphe — APPROVED** [review]
LGTM!! :tada:

**Comment by MarinhoFeliphe** [comment]
ci:forward:force

**Comment by mateussgn** [comment]
ci:forward:force

**Comment by pedro-oliveira446** [comment]
That pr will be resent when [this one](https://github.com/brianchandotcom/liferay-portal/pull/137612) is merged.

**Pull request closed without merging**